app/reducers/activity.js

```diff
@@ -336,11 +337,21 @@ const pendingActivity = createSelector(
   invoices => groupAll(invoices.filter(invoice => !invoice.settled && !invoiceExpired(invoice)))
 )
 
+const internalActivity = createSelector(
+  transactionsSelector,
+  transactions => {
+    return groupAll([
+      ...transactions.filter(transaction => transaction.isFunding || transaction.isClosing)
```

Comment:
Is there a need to do a spread since `filter` already returns a new array?

Reply:
Good catch. You;'re right. I have updated this.